fem/bilininteg.hpp

```diff
@@ -159,6 +159,15 @@ class BilinearFormIntegrator : public NonlinearFormIntegrator
                                    FaceElementTransformations &Trans,
                                    DenseMatrix &elmat);
 
+   /** Abstract method used for assembling TraceFaceIntegrators for
+       DPG weak formulations. */
```

Comment:
DPG is mentioned in a few comments, but is there a need to specify DPG? Couldn't the integrators be used more generally?

Reply:
@dylan-copeland, note that the signature is slightly different than our usual integrators. For now, these "DPG" integrators cannot be used by the standard `(Mixed)BilinearForm`